### DISCUSSION TITLE
button-variant mixin in master is older than released version

### DISCUSSION THREAD

**Discussion by jeffmarshall** [twbs/bootstrap#30832] [General] [answered]
I'm looking forward to using the `$color` parameter in the `button-variant` mixin here: https://github.com/twbs/bootstrap/blob/master/scss/mixins/_buttons.scss#L9

It was last committed over a month ago, but this week's release of 4.5.0 doesn't include the new parameter: https://github.com/twbs/bootstrap/blob/v4.5.0/scss/mixins/_buttons.scss

Is this deliberate?

**Comment by mdo** [comment, marked as answer]
`master` is our upcoming v5, `v4-dev` is for v4. v5 has a ton of changes that won't ever come to v4, hence the differences you're seeing.

**Reply by jeffmarshall** [reply]
Ohhh

Thanks :D
